test/api/run

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+import sys
+
+# Use the run.py library from ../cql-pytest:
+sys.path.insert(1, sys.path[0] + '/../cql-pytest')
+import run
+
+print('Scylla is: ' + run.scylla + '.')
+
+ssl = '--ssl' in sys.argv
+if ssl:
+    cmd = run.run_scylla_ssl_cql_cmd
+    check_cql = run.check_ssl_cql
+else:
+    cmd = run.run_scylla_cmd
+    check_cql = run.check_cql
+
+pid = run.run_with_temporary_dir(cmd)
+ip = run.pid_to_ip(pid)
+
+run.wait_for_cql(pid, ip)
```

Comment:
Although this will work in practice - because we (?) start the REST API before CQL (do we?) - the better approach would be to also check the REST API came up. You can see an example of how to check both CQL and something else in the new patch I sent for redis. Or in the already existing test/alternator/run.

Reply:
We do start listening on the API before CQL (one reason is to be able to stop reshape compaction when populating tables in the distributed_loader), but it will make no harm if we check for both CQL and REST API...